samples/dotnet/Directory.Build.props

```diff
@@ -0,0 +1,23 @@
+﻿<Project>
+  <PropertyGroup>
+    <!-- Default properties inherited by all projects. Projects can override. -->
+    <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <AnalysisLevel>latest</AnalysisLevel>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <LangVersion>10</LangVersion>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>disable</ImplicitUsings>
+    <IsPackable>disable</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
```

Comment:
So you want your PDB's to be in Windows format when we are in release and in multi-platform format when we are in debug???

I'm not sure I understand the intent here...

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation

Reply:
Good question. This was copied from the other project, so there's no change here... but that doesn't necessarily mean it was correct before, or should be ported. I'll read the docs.

Reply:
Here's my understanding... the reason we have DebugType=full (and any debugging symbols at all in 'Release') may be that the dotnet analyzers use them, and we've turned on dotnet format to run in Release builds only.

Some of the analyzers cannot use the "portable" format, hence the difference from 'Debug' builds:
> Portable PDBs can be read on any operating system, but there are a number of places where they aren't supported yet. Here are a few –
> ...
> - C# Code analysis (aka FxCop), note that this doesn't apply to Roslyn Analyzer.

This is consistent with the SK dotnet sdk: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/Directory.Build.props

And the CopilotChatWebApi: https://github.com/microsoft/semantic-kernel/blob/main/samples/apps/copilot-chat-app/webapi/CopilotChatWebApi.csproj

Reply:
I must confess I'm not sure what to think about this. But don't let this block this PR. I mean, it worked before so it should work now too. I'm just a bit confused about this. Then again, I'm confused about many things in this world.

Reply:
Update: I've been testing various dotnet format violations without the above tags on release, and it doesn't seem to make any difference. That leads to second theory, which is that these may have been included to ensure the symbols made it into (released) nuget packages -- which of course does not apply to samples. I've removed that whole chunk from both this project and copilot chat, and I'll experiment further with sdk projects (ensure that .snupkg files are generated properly).